python/ray/serve/built_application.py

```diff
@@ -23,7 +23,7 @@ def __setitem__(self, *args):
 
 
 @DeveloperAPI
-class Application:
+class BuiltApplication:
```

Comment:
FYI, I think you need to manually insert these classes into Sphinx now or else they won't get rendered.

Reply:
going to clean up API ref & add this in a separate PR